TESTS/netsocket/README.md

```diff
@@ -58,17 +58,11 @@ echo.mbedcloudtesting.com has IPv6 address 2a05:d018:21f:3800:8584:60f8:bc9f:e61
 
 **Open services in the test server**
 
--   Echo protocol, [RFC 862](https://tools.ietf.org/html/rfc862) is
-    enabled in both TCP and UDP. Port 7. Port 2007 for TLS.
--   Discard protocol, [RFC 863](https://tools.ietf.org/html/rfc863) is
-    enabled in both TCP and UDP. Port 9. Port 2009 for TLS.
--   Character generator protocol, [RFC 864](https://tools.ietf.org/html/rfc864) is
-    enabled in both TCP and UDP. Port 19. Output pattern should follow
-    the proposed example pattern in RFC.
--   Daytime protocol, [RFC 867](https://tools.ietf.org/html/rfc867) in
-    both TCP and UDP. Port 13. Port 2013 for TLS.
--   Time protocol, [RFC 868](https://tools.ietf.org/html/rfc868) in
-    both TCP and UDP. Port 37.
+-   Echo protocol, [RFC 862](https://tools.ietf.org/html/rfc862) is enabled in both TCP and UDP. Port 7. Port 2007 for TLS.
```

Comment:
Maybe change the to `is enabled on both TCP and UDP on port 7. Port 2007 for TLS`